source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -4840,10 +4839,10 @@ CK_DECLARE_FUNCTION( CK_RV, C_Verify )( CK_SESSION_HANDLE hSession,
         }
         else if( pxSessionObj->xOperationVerifyMechanism == CKM_AES_CMAC )
         {
-            if( ulSignatureLen != MBEDTLS_AES_BLOCK_SIZE )
+            if( ulSignatureLen != PKCS11_AES_CMAC_MIN_SIZE )
```

Comment:
Is this change just for MISRA or is this a bugfix?

Reply:
This is just for MISRA, the existing is a macro defined by mbedtls which is `16` the new is a macro we already defined for the AES CMAC block length

Reply:
Our's is `16UL` so MISRA doesn't complain.